src/components/migrationToV3.stories.mdx

```diff
@@ -437,6 +442,140 @@ When you implement your own theme, you need to use `globalStyle` instead of `glo
 
 ## Device
 
+### Renaming
+
+Below table summarizes all renaming of device management related stuff. Simply use new naming to replace old naming for migration.
+
+| Before in 2.x                                               | After in 3.x                                               |
+| ----------------------------------------------------------- | ---------------------------------------------------------- |
+| `DevicePermissionStatus`                                    | `DeviceLabelTriggerStatus`                                 |
+| `DevicePermissionStatus.UNSET`                              | `DevicePermissionStatus.UNTRIGGERED`                       |
+| `useDevicePermissionStatus`                                 | `useDeviceLabelTriggerStatus`                              |
+|                                                             |                                                            |
+| `MeetingManager.devicePermissionStatus`                     | `MeetingManager.deviceLabelTriggerStatus`                  |
+|                                                             |                                                            |
+| `MeetingManager.selectAudioInputDevice()`                   | `MeetingManager.startAudioInputDevice()`                   |
+| `MeetingManager.selectAudioOutputDevice()`                  | `MeetingManager.startAudioOutputDevice()`                  |
+| `MeetingManager.selectVideoInputDevice()`                   | `MeetingManager.startVideoInputDevice()`                   |
+|                                                             |                                                            |
+| `MeetingManager.devicePermissionsObservers`                 | `MeetingManager.deviceLabelTriggerStatusObservers`         |
+| `MeetingManager.subscribeToDevicePermissionStatus()`        | `MeetingManager.subscribeToDeviceLabelTriggerStatus()`     |
+| `MeetingManager.unsubscribeFromDevicePermissionStatus()`    | `MeetingManager.unsubscribeFromDeviceLabelTriggerStatus()` |
+|                                                             |                                                            |
+| `MeetingManager.deviceLabelTriggerChangeObservers`          | `MeetingManager.deviceLabelTriggerObservers`               |
+| `MeetingManageer.subscribeToDeviceLabelTriggerChange()`     | `MeetingManageer.subscribeToDeviceLabelTrigger()`          |
+| `MeetingManageer.unsubscribeFromDeviceLabelTriggerChange()` | `MeetingManageer.unsubscribeFromDeviceLabelTrigger()`      |
+
+### Update Type of Selected Device in MeetingManager
+
+In V3, we improve type for selected device in `MeetingManager` and update the corresponding components and hooks.
```

Comment:
Suggest re-phrasing the sentence: 
In V3, we have improved the `type` for selected device in `MeetingManager` and corresponding components and hooks.

Reply:
Updated